utils/python/CIME/case.py

```diff
@@ -48,7 +51,7 @@ def flush(self):
 
     def get_value(self, item, attribute={}, resolved=True, subgroup=None):
         result = None
-        for env_file in self._env_files:
+        for env_file in self._env_entryid_files:
```

Comment:
TBD,  there are specific interfaces to read these files but currently no way to change them except with a text editor